test/test_functions.py

```diff
@@ -804,6 +804,16 @@ def test_make_bag(self):
             Query().summarize(f.make_bag(t.stringField, t.numField)).render()
         )
 
+    def test_make_bag_if(self):
+        self.assertEqual(
+            " | summarize make_bag_if(stringField, numField == 1)",
+            Query().summarize(f.make_bag_if(t.stringField, t.numField == 1)).render()
+        )
+        self.assertEqual(
+            " | summarize make_bag_if(stringField, numField == 1, numField)",
```

Comment:
Let's change the `max size` argument to an explicit integer of your choice. 
A value from the DB can work, but it doesn't reflect the common expected usage.

Reply:
This comment applies also to the `make_list_if` and `make_set_if`